src/components/Profile/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react'
+import { BasicProfileIcon } from '@/assets/icons'
+
+interface ProfileProps {
+  image?: string | ReactNode
```

Comment:
이건 그냥 ReactNode로 적어도 string도 포함됩니다!!

```suggestion
  image?: ReactNode
```

Reply:
그런데 사실 3이나 다른 타입들의 데이터를 전달해도 에러가 나지 않아요 .. 이유는 [`ReactNode`](https://www.howdy-mj.me/react/react-node-and-jsx-element)를 사용했기 때문이죠!
이걸 방지하려면 string | ReactElement 혹은 string | JSX.Element 를 사용할 수 있을 것 같아요!